webpack.config.js

```diff
@@ -0,0 +1,49 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const ESLintPlugin = require('eslint-webpack-plugin');
+// const StylelintPlugin = require('stylelint-webpack-plugin');
```

Comment:
- You made a great job so far , but you committed old pieces of code as inline comments. They will make your project look messy. Kindly improve that by deleting these line 👌